cleanco/termdata.py

```diff
@@ -1,145 +1,1047 @@
-   'Sole Proprietorship': ['e.u.', 's.p.', 't:mi', 'tmi', 'e.v.', 'e.c.', 'et', 'obrt',
-      'fie', 'ij', 'fop', 'xt'
-   ]
+    "Additional Liability Company": [
+        "товариство з додатковою відповідальністю",
+        "tovarystvo z dodatkvoiu vidpovidalnistiu",
+    ],
+    "Assocation of Co-Owners": [
+        "vereniging van mede eigenaars van",
+        "association des coproprietaires de",
+        "společenství vlastníků jednotek",
+        "syndicat de la copropriété du",
+        "associação comunitária dos moradores",
+    ],
+    "Cooperative Company with Limited Liability": [
+        "eingetragene genossenschaft mit beschrankter haftpflicht",
+    ],
+    "Corporation": [
+        "company",
+        "incorporated",
+        "corporation",
+        "corp.",
+        "corp",
+        "inc",
+        "& co.",
+        "& co",
+        "inc.",
+        "s.p.a.",
+        "n.v.",
+        "a.g.",
+        "ag",
+        "nuf",
+        "s.a.",
+        "s.f.",
+        "oao",
+        "co.",
+        "co",
+    ],
+    "General Partnership": [
+        "soc.col.",
+        "stg",
+        "d.n.o.",
+        "ltda.",
+        "v.o.s.",
+        "a spol.",
+        "ve\xc5\x99. obch. spol.",
+        "kgaa",
+        "o.e.",
+        "s.f.",
+        "s.n.c.",
+        "s.a.p.a.",
+        "j.t.d.",
+        "v.o.f.",
+        "sp.j.",
+        "og",
+        "sd",
+        " i/s",
+        "ay",
+        "snc",
+        "oe",
+        "bt.",
+        "s.s.",
+        "mb",
+        "ans",
+        "da",
+        "o.d.",
+        "hb",
+        "pt",
+    ],
+    "Joint Stock / Unlimited": [
+        "unltd",
+        "ultd",
+        "sal",
+        "unlimited",
+        "saog",
+        "saoc",
+        "aj",
+        "yoaj",
+        "oaj",
+        "akc. spol.",
+        "a.s.",
+        "akciová společnost",
+        "aktiengesellschaft",
+    ],
+    "Joint Venture": ["esv", "gie", "kv.", "qk"],
+    "Limited": [
+        "pty. ltd.",
+        "pty ltd",
+        "ltd",
+        "l.t.d.",
+        "bvba",
+        "d.o.o.",
+        "ltda",
+        "gmbh",
+        "g.m.b.h",
+        "gesellschaft mit beschränkter haftung",
+        "kft.",
+        "kht.",
+        "zrt.",
+        "ehf.",
+        "s.a.r.l.",
+        "d.o.o.e.l.",
+        "s. de r.l.",
+        "s. de r.l. de c.v.",
+        "b.v.",
+        "tapui",
+        "sp. z.o.o.",
+        "sp. z o.o.",
+        "spółka z o.o.",
+        "spółka z ograniczoną odpowiedzialnością",
+        "s.r.l.",
+        "s.l.",
+        "s.l.n.e.",
+        "ood",
+        "oy",
+        "rt.",
+        "teo",
+        "uab",
+        "scs",
+        "sprl",
+        "limited",
+        "bhd.",
+        "sdn. bhd.",
+        "sdn bhd",
+        "as",
+        "lda.",
+        "tov",
+        "pp",
+        "товариство з обмеженою відповідальністю",
+        "tovarystvo z obmezhenoiu vidpovidalnistiu",
+    ],
+    "Limited Liability Company": [
+        "pllc",
```

Comment:
This is also in Professional Limited Liability Company. I assume this would be removed?